libcontainer/init_linux.go

```diff
@@ -499,6 +499,26 @@ func signalAllProcesses(m cgroups.Manager, s os.Signal) error {
 	if err := m.Freeze(configs.Frozen); err != nil {
 		logrus.Warn(err)
 	}
+	// If signal is SIGKILL try manager.Kill()
+	// inorder to leverage "cgroup.kill" instead
+	// of iterating over each process. "cgroup.kill"
+	// was added in Linux kernel 5.14 onwards read
+	// more about it https://lwn.net/Articles/855924/
```

Comment:
You don't need to add a reference to LWN every time the feature is used (in fact the reference isn't really necessary at all -- `cgroup.kill` is a fairly easy-to-understand part of the kernel ABI that is documented in the actual kernel docs now).

Reply:
@cyphar thanks removed in latest commit